Rename gradle functions (#11)
unimodules/react-native-unimodules: README.md

### Configure Android

> - Add `apply from: '../node_modules/react-native-unimodules/gradle.groovy'` and then `includeUnimoduleProjects()` to the top of  `android/settings.gradle`

Reviewer: ```suggestion
- Add `apply from: '../node_modules/react-native-unimodules/gradle.groovy'` and then `includeUnimodulesProjects()` to the top of  `android/settings.gradle`
```

> - Add `apply from: '../../node_modules/react-native-unimodules/gradle.groovy'` anywhere in `android/app/build.gradle` and then `addUnimoduleDependencies()` inside `dependencies {}` block.

Reviewer: ```suggestion
- Add `apply from: '../../node_modules/react-native-unimodules/gradle.groovy'` anywhere in `android/app/build.gradle` and then `addUnimodulesDependencies()` inside `dependencies {}` block.
```

> - If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param `modulesPaths` for both of these functions: `includeUnimoduleProjects([modulesPaths: ['./path/to/node_modules']])`, `addUnimoduleProjects([modulesPaths: ['./path/to/node_modules']])`

Reviewer: ```suggestion
- If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param `modulesPaths` for both of these functions: `includeUnimodulesProjects([modulesPaths: ['./path/to/node_modules']])`, `addUnimodulesDependencies([modulesPaths: ['./path/to/node_modules']])`
- You can also customize the configuration of the unimodules dependencies (the default is `implementation`, if you're using Gradle older than 3.0, you will need to set `configuration: "compile"` in `addUnimodulesDependencies`, like: `addUnimodulesDependencies([configuration: "compile"])`)
```
